.github/workflows/manual-analysis.yml

```
@@ -0,0 +1,62 @@
+    - name: Run TradingAgents Analysis
+      env:
+        OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        FINNHUB_API_KEY: ${{ secrets.FINNHUB_API_KEY }}
+        TRADINGAGENTS_TICKER: ${{ github.event.inputs.ticker }}
+        TRADINGAGENTS_DATE: ${{ steps.set_date.outputs.date }}
+        TRADINGAGENTS_OUTPUT_PATH: results/github-actions/analysis-${{ github.run_number }}.md
+      run: |
+        python cli/non_interactive.py
+        
+    - name: Upload analysis results
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)